src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java

```diff
@@ -19,10 +18,7 @@
 
   @Test(groups = "mercury_recentWikiActivity_anonNavigateToRWAfromMenu")
   public void mercury_recentWikiActivity_anonNavigateToRWAfromMenu() {
-    /**
-     * Temporary change until the end of experiment (21.04.2016) described in DAT-4052.
-     */
-    new Navigate(driver).toPage("/wiki/Aga_Wikia");
+    new MainPage().open();
```

Comment:
please restore it, as it was https://github.com/Wikia/selenium-tests/blob/36c8ed38c3c2347add58c6ac495eb2401343a80e/src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java#L21